includes/wikia/parser/templatetypes/handlers/ScrollboxTemplate.class.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+class ScrollboxTemplate {
+	/**
+	 * @desc gets the longest value from template arguments
```

Comment:
This is basically getLongestElement from $stringArr
What do you think about naming it in a more general way so it can be reused if needed?